src/index.html

```diff
@@ -4,15 +4,63 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
-    <title>Stars</title>
+    <title>Stars Rating</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
```

Comment:
The attributes of the link tag should also be formatted with each attribute on a new line, following the same indentation rules as the meta viewport tag.